repo/overrides/default.nix

```diff
@@ -134,6 +139,10 @@ in
 		solo5-kernel-vertio = disableStackProtection opamPackages.solo5-kernel-vertio;
 		solo5-kernel-ukvm = disableStackProtection opamPackages.solo5-kernel-ukvm;
 
+		sqlite3 = overrideAll (impl: {
+			buildInputs = impl.buildInputs ++ [ sqlite ];
```

Comment:
`sqlite3` opam package depends on `conf-sqlite3`, so this seems like it shouldn't be necessary.